setup.py

```diff
@@ -69,15 +71,23 @@ class MyBuildExt(build_ext):
         # this list should be carefully examined when pyarrow lib is
         # upgraded
         arrow_libs_to_copy = {
-            "linux": ["libarrow.so.600", "libarrow_python.so.600"],
-            "darwin": ["libarrow.600.dylib", "libarrow_python.600.dylib"],
-            "win32": ["arrow.dll", "arrow_python.dll"],
+            "linux": ["libarrow.so.800", "libarrow_python.so.800", "libparquet.so.800"],
```

Comment:
Just curious: Since this is only in `setup.py`, does this mean at setup time we require 8.0.0 to be available, but not really at runtime?